README.md

```diff
@@ -1 +1,31 @@
-# Vet-clinic
\ No newline at end of file
+# Vet-clinic
+In this projects, a relational database is used to create the data structure for a vet clinic complete database with data about:
+
+- animals
+- animals' owners
+- clinic employees
+- visits
+
+## Built With
+postgreSQL
+
+## Author
+**Zahra**
+- GitHub: [@githubhandle](https://github.com/ZahraArshia)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zahra-arshia/)
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly add the `Getting Started` section and include the Getting started from the [template](https://github.com/microverseinc/curriculum-template-databases) for this project.

- [ ] Also add the `Acknowledgments` section and fix the `MIT` License which shows a 404 error page so that your repo looks more profesional.